tests/aviftest.c

```diff
@@ -467,14 +299,10 @@ int main(int argc, char * argv[])
     // Parse cmdline
     for (int i = 1; i < argc; ++i) {
         char * arg = argv[i];
-        if (!strcmp(arg, "-g")) {
-            generate = AVIF_TRUE;
-        } else if (!strcmp(arg, "--io-only")) {
-            ioOnly = AVIF_TRUE;
+        if (!strcmp(arg, "--io-only")) {
+            // Deprecated, do nothing
```

Comment:
Should we print a warning message that says this option is deprecated and should be removed?

Reply:
Done!